content/applications/inventory_and_mrp/manufacturing/shop_floor/shop_floor_overview.rst

```diff
@@ -0,0 +1,204 @@
+===================
+Shop Floor Overview
+===================
+
+.. _manufacturing/shop_floor/shop_floor_overview:
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+.. |MOs| replace:: :abbr:`MOs (Manufacturing Orders)`
+
+The *Shop Floor* module is a companion module to the *Manufacturing* app. *Shop Floor* provides a
+visual interface for processing manufacturing orders (MOs) and work orders. It also allows
+manufacturing employees to track the amount of time spent working on manufacturing and work orders.
+
+The *Shop Floor* module is installed alongside the *Manufacturing* app. It cannot be installed by
+itself. To install the *Manufacturing* app, navigate to :menuselection:`Apps`, search for
+"manufacturing" in the :guilabel:`Search...` bar, and then click :guilabel:`Install` on the
+:guilabel:`Manufacturing` app card.
+
+.. important::
+   The *Shop Floor* module replaces the tablet view functionality of the *Manufacturing* app, and is
+   only available in Odoo versions 16.4 and later.
+
+   To check the version number of an Odoo database, navigate to :menuselection:`Settings` and scroll
+   down to the :guilabel:`About` section at the bottom of the page. The version number is displayed
+   there.
+
+   To switch to a newer version of Odoo, see the documentation on :ref:`upgrading a database
+   <administration/upgrade>`.
+
+Navigation
+==========
+
+Shop Floor is broken down into three main views, which can be selected from the navigation bar at
+the top of the module:
+
+- The :guilabel:`All` page serves as the main dashboard for the module and displays information
+  cards for |MOs|.
+- Each work center also has a dedicated page which shows information cards for work orders assigned
+  to that work center. Work center pages can be toggled on or off by clicking the :guilabel:`+
+  (plus)` button in the navigation bar, selecting or deselecting them on the pop-up window that
+  appears, and then clicking :guilabel:`Confirm`.
+- The :guilabel:`My` page shows information cards for all work orders assigned to the employee whose
+  profile is currently active in the operator panel on the left side of the module. Other than only
+  showing work orders assigned to the active employee, this page functions the same as the pages for
+  each work center.
+
```

Comment:
oh no i just realized i opened the runbot for 17.0 to take this screenshot! make sure you take a different screenshot in 16.4 because the names of the menus are different:
16.4: "All" page and "My" page
17.0: "All MO" and "My WO" page
![image](https://github.com/odoo/documentation/assets/23611711/1fbc347d-2050-47ce-8495-f8bc223eb627)

![image](https://github.com/odoo/documentation/assets/23611711/c095c1ef-d2eb-480b-9d68-61fac34bdf5d)